python/hail/context.py

```diff
@@ -465,6 +471,74 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
         return VariantDataset(self, jvds)
 
     @handle_py4j
+    def import_vcf_generic(self, path, force=False, force_bgz=False, header_file=None, npartitions=None,
```

Comment:
We could do that -- my original thought was the documentation was a bit overwhelming with both cases combined. But you're right that it would be better.